core/src/options/input_variant.rs

```diff
@@ -11,6 +12,7 @@ pub struct InputVariant {
     attr_name: Option<String>,
     data: Fields<InputField>,
     skip: Option<bool>,
+    pub word: Option<SpannedValue<bool>>,
```

Comment:
```suggestion
    /// Whether or not the variant should be used to create an instance for
    /// `FromMeta::from_word`.
    pub word: Option<SpannedValue<bool>>,
```

Copying the doc comment over from the codegen field so people who are reading the `options` module to see what darling options are available can figure out why this exists.